src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -147,10 +148,8 @@ private void testActions() throws Exception {
                                          spa.gender, spa.moreInfo);
         profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Usual Nationality",
                                           "female", "this is enough!$%&*</>");
-        profilePage.verifyStatus(StringHelper.toString(spa.getInvalidityInfo(), " ")
-                                             // de-sanitize
-                                             .replace("&lt;", "<").replace("&gt;", ">")
-                                             .replace("&quot;", "\"").replace("&#x2f;", "/"));
```

Comment:
Sanitization no longer needed because invalid input not displayed as part of error messages.